client/app/models/contact.coffee

```diff
@@ -148,6 +148,26 @@ module.exports = class Contact extends Backbone.Model
         _.compact([pf, fn, mn, gn, sf]).join ' '
 
 
+    # We override toJSON to ensure the exported object is the a true object.
+    # Allow a options.toVCF boolean to sanitize the output before.
+    toJSON: ({toVCF} = {}) ->
```

Comment:
Technically, we don't change its signature, as the [`Model.toJSON`](http://backbonejs.org/#Model-toJSON) method declares an option hash which is, in fact, never used x).

By overridding it here, we also ensure datapoints are properly JSONified every time `toJSON` is called, which seems to be a safer way to deal with it, isn't it?